integration/tests/bft/admin/blocklist/admin_command_blocklist_test.go

```diff
@@ -0,0 +1,46 @@
+package blocklist
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+type AdminCommandBlockListTestSuite struct {
+	Suite
+}
+
+func TestAdminCommandBlockList(t *testing.T) {
+	suite.Run(t, new(AdminCommandBlockListTestSuite))
+}
+
+// TestAdminCommandBlockList ensures that the blocklist admin command works as expected. When a node is blocked via the admin blocklist command
+// the libp2p connection to that node should be pruned immediately and the connection should start to block incoming connection request. This test
+// sets up 2 corrupt nodes a sender and receiver, the sender will send messages before and after being blocked by the receiver node via
+// the blocklist admin command. The receiver node is expected to receive messages like normal before blocking the sender, after blocking the sender
+// it should not receive any messages.
+func (a *AdminCommandBlockListTestSuite) TestAdminCommandBlockList() {
+	// send some authorized messages indicating the network is working as expected
+	a.Orchestrator.sendAuthorizedMsgs(a.T())
+	unittest.RequireReturnsBefore(a.T(), a.Orchestrator.authorizedEventsReceivedWg.Wait, 5*time.Second, "could not send authorized messages on time")
+	// messages with correct message signatures are expected to always pass libp2p signature verification and be delivered to the victim EN.
+	require.Equal(a.T(), int64(numOfAuthorizedEvents), a.Orchestrator.authorizedEventsReceived.Load(), fmt.Sprintf("expected to receive %d authorized events got: %d", numOfAuthorizedEvents, a.Orchestrator.unauthorizedEventsReceived.Load()))
+
+	// after blocking node a.senderVN we should not receive any messages from that node. We wait for
+	// 500 milliseconds to reduce the small chance of a race condition between the time a node is blocked
+	// and the time the blocked node sends the first unauthorized message
+	a.blockNode(a.senderVN)
+	time.Sleep(500 * time.Millisecond)
+
+	// send unauthorized messages and sleep for 3 seconds to allow all requests to be processed
+	a.Orchestrator.sendUnauthorizedMsgs(a.T())
```

Comment:
These messages are labeled as "unauthorized" because they are sent after the node is blocked, it doesnt imply that the messages break some authorization rules. I renamed from "unauthorized -> expectedBlockedMsgs https://github.com/onflow/flow-go/pull/3877/commits/3f3eeec076db8a44e172aab45acecd932bcad8fa

Reply:
Please also extend the `godoc` saying that in normal situations if the node is not block listed, all these messages would be considered legit and hence would be delivered to the recipients.